dutchx_scripts/daoCreateDutchX.ts

```diff
@@ -0,0 +1,227 @@
+       */
+      priceOracleInterfaceAddress = "0x1C54f6146bA3656739A5c0781Cc054FA0C3951C1";
+
+      /**
+       * TODO: use mock instead of hardwired above
+       */
+      // priceOracleInterfaceMock =
+      //   await contractNew(web3, networkName, { name: "PriceOracleMock" }, "max", "50000000000") as any;
+
+      // priceOracleInterfaceAddress = priceOracleInterfaceMock.address;
+
+      // await priceOracleInterfaceMock.setTokenPrice("0xc778417e063141139fce010982780140aa0cd5ab", 380407, 200000000);
+      // await priceOracleInterfaceMock.setTokenPrice("0x3615757011112560521536258c1e7325ae3b48ae", 380407, 200000000);
+      // await priceOracleInterfaceMock.setTokenPrice("0x00df91984582e6e96288307e9c2f20b38c8fece9", 380407, 200000000);
+      // await priceOracleInterfaceMock.setTokenPrice("0xa1f34744c80e7a9019a5cd2bf697f13df00f9773", 380407, 200000000);
+      // await priceOracleInterfaceMock.setTokenPrice("0x543Ff227F64Aa17eA132Bf9886cAb5DB55DCAddf", 380407, 200000000);
+      break;
+    case "Live":
+      externalTokenLockerAddress = "";
+      gnoTokenAddress = "0x6810e776880c02933d47db1b9fc05908e5386b96";
+      priceOracleInterfaceAddress = "";
+      break;
+  }
+
+  console.log(`gnoTokenAddress: ${gnoTokenAddress}`);
+  console.log(`genTokenAddress: ${genTokenAddress}`);
+
+  daoSchema.schemes = [...(daoSchema.schemes || []), ...
+    [
+      {
+        address: lockingEth4Reputation.address,
+        name: "LockingEth4Reputation",
+      },
+      {
+        address: externalLocking4Reputation.address,
+        name: "ExternalLocking4Reputation",
+      },
+      {
+        address: lockingToken4Reputation.address,
+        name: "LockingToken4Reputation",
+      },
+      {
+        address: auction4Reputation.address,
+        name: "Auction4Reputation",
+      },
+    ]];
+
+  const dao = (await daoCreate(web3, networkName, daoSchema, "true")) as DAO;
+
+  /**********************
+   !!!! Start and end dates should be such that they can be divided evenly by (NUM_AUCTIONS / 1000)
+   **********************/
+  const lockingPeriodStartDate      = new Date("2019-01-15T15:00:00.000+0200");
+  const lockingPeriodEndDate        = new Date("2019-01-15T17:00:00.000+0200");
+  const lockingPeriodStartDate_mgn  = new Date("2019-01-15T16:30:00.000+0200");
+  const lockingPeriodEndDate_mgn    = new Date("2019-01-15T17:00:00.000+0200");
+  /**********************
+   !!!! Should be a number of auctions such that the sum of auction rep comes out to exactly TOTAL_REP_REWARD * AUCTION_BIDDING_RATIO
+   **********************/
+  const NUM_AUCTIONS = 5;
+  const MAX_LOCK_PERIOD = 7200; // 2 hours in seconds, use 31536000 for one year
+  /**********************
+   !!!! Should be an amount that yields a whole number when multiplied by any of the ratios below
+   **********************/
+  const TOTAL_REP_REWARD = 1000000000;
```

Comment:
This is 1000 million
it should be 100 million

Reply:
https://github.com/daostack/dx-deployment-script/pull/1/commits/4a606d620bab8af0d3923dbc470b75b855f15642